queries.sql

```diff
@@ -1,3 +1,15 @@
-/*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals WHERE name LIKE '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE date_of_birth >='2016/01/01' AND date_of_birth <='2019/12/31';
+
+SELECT name FROM animals WHERE neutered=true AND escape_attempts<3;
+
+SELECT date_of_birth FROM animals WHERE name='Agumon' OR name='Pikachu';
+
+SELECT name,escape_attempts FROM animals WHERE weight_kg>10.5;
+
+SELECT * FROM animals WHERE neutered=true;
+
+SELECT * FROM animals WHERE name!='Gabumon';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
- [ ] As it was required of you in the project requirements, you are required to add the screenshots of the query results in your pull request description. Kindly include it.

![image](https://user-images.githubusercontent.com/86318284/215990702-7f6e08b6-aed6-49db-9d73-ec51a2fca7b4.png)